### PR TITLE
[#AIO-964] Fixes previous step bug for group steps

### PR DESCRIPTION
**Pull request by aravindpudayoor**

### CODE REVIEW

File: app/presenters/nfg_onboarder/group_steps_navigation_bar_presenter.rb [hunk 1]
```diff
@@ -44,11 +44,35 @@ def visited?(nav_step)
...
-      h.controller.class.name.underscore.split('/')[1]
+      h.controller.class.name.underscore.split('/')
+    end
+
+    # previous_wizard_path is the last step that was visited in the group. It is used to
+    # determine the previous step but it is not the actual previous step when it comes to
+    # the first step of the group. So we need to check whether the active step is the first step
+    # of the group step or not. If it is the first step of the group, we need to return
+    # the previous_group_path. Otherwise, we need to return the previous_wizard_path.
+    def previous_path
```
Comment (kcpavan) [inline app/presenters/nfg_onboarder/group_steps_navigation_bar_presenter.rb:59]: `previous_wizard_or_group_path` may be a more appropriate name for this method.

**Review by kcpavan — APPROVED** [review]
Looks good

**Comment by kylejmorin** [comment]
@aravindpudayoor is this good to merge in now?

**Comment by aravindpudayoor** [comment]
@kylejmorin Yes, this is good to merge.

**Pull request merged**